api/resources/translations/messages-es.properties

```diff
@@ -391,8 +391,8 @@ child_birth_date = Fecha nacimiento niño
 child_birth_outcome = Resultado del parto
 child_birth_weight = Peso del niño al nacer
 clear = borrar
-cleared = Marca de Tiempo Borrada
-clientDdoc.version = Versión de Cliente DDoc
+cleared = Marca de tiempo borrada
+clientDdoc.version = Versión de cliente DDoc
```

Comment:
```suggestion
clientDdoc.version = Revisión del cliente webapp
```
Or `Número de revisión en el cliente webapp`, but it's long. This data is targeting system admins. It's okay to use `webapp`

<img width="294" alt="Screenshot 2024-02-20 at 6 23 30 PM" src="https://github.com/medic/cht-core/assets/66472237/7d8acd87-ad1b-4014-a60c-7d4554a6c604">